### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by jeanniefinks**
various grammatical edits

### CODE REVIEW

**Review by bfineran — DISMISSED** [review]
Thanks @jeanniefinks

File: examples/aws-sagemaker/README.md [hunk 1]
```diff
@@ -14,19 +14,19 @@ See the License for the specific language governing permissions and
 limitations under the License.
 -->
 
-# Deploy DeepSparse with Amazon SageMaker
+# Deploying DeepSparse with Amazon SageMaker
 
 [Amazon SageMaker](https://docs.aws.amazon.com/sagemaker/index.html)
-offers easy to use infrastructure for deploying deep learning models at scale.
+offers easy-to-use infrastructure for deploying deep learning models at scale.
```
Comment (rahul-tuli) [inline examples/aws-sagemaker/README.md:20]:
```suggestion
offers an easy-to-use infrastructure for deploying deep learning models at scale.
```

File: examples/aws-sagemaker/README.md [hunk 2]
```diff
@@ -36,11 +36,11 @@ with `deepsparse` installed and creates an executable command `serve` that runs
...
 
 For general customization of the server, changes should not need to be made
-to the dockerfile, but to the `config.yaml` file that the dockerfile reads from
+to the dockerfile, but to the `config.yaml` file that the Dockerfile reads from
 instead.
 
 ### config.yaml
-`config.yaml` used to configure the DeepSparse serve running in the Dockerfile.
+`config.yaml` is used to configure the DeepSparse server running in the Dockerfile.
 It is important that the config contains the line `integration: sagemaker` so
```
Comment (rahul-tuli) [inline examples/aws-sagemaker/README.md:44]:
```suggestion
The config must contain the line `integration: sagemaker` so
```

File: examples/aws-sagemaker/README.md [hunk 3]
```diff
@@ -36,11 +36,11 @@ with `deepsparse` installed and creates an executable command `serve` that runs
 `invocations/` endpoint.
```
Comment (rahul-tuli) [inline examples/aws-sagemaker/README.md:36]: This comment actually refers to line 34:
Change Double space in front of Sagemaker to a Single Space;

i.e from `on port 8080.  SageMaker will execute this image by running`
To: `on port 8080. SageMaker will execute this image by running`

File: examples/aws-sagemaker/README.md [hunk 4]
```diff
@@ -14,19 +14,19 @@ See the License for the specific language governing permissions and
...
-# Deploy DeepSparse with Amazon SageMaker
+# Deploying DeepSparse with Amazon SageMaker
 
 [Amazon SageMaker](https://docs.aws.amazon.com/sagemaker/index.html)
-offers easy to use infrastructure for deploying deep learning models at scale.
+offers easy-to-use infrastructure for deploying deep learning models at scale.
 This directory provides a guided example for deploying a 
 [DeepSparse](https://github.com/neuralmagic/deepsparse) inference server on SageMaker.
 Using both of these tools, deployments benefit from sparse-CPU acceleration from
 DeepSparse and automatic scaling from SageMaker.
```
Comment (rahul-tuli) [inline examples/aws-sagemaker/README.md:24]: I suggest converting this from passive to active

```suggestion
Deployments benefit from both sparse-CPU acceleration with
DeepSparse and automatic scaling from SageMaker.
```

File: examples/aws-sagemaker/README.md [hunk 5]
```diff
@@ -255,7 +255,7 @@ sm_boto3.delete_model(ModelName=model_name)
 
 ## Next Steps
 These steps create an invokable SageMaker inference endpoint powered with the DeepSparse
```
Comment (rahul-tuli) [inline examples/aws-sagemaker/README.md:257]:
```suggestion
These steps create an invokable SageMaker inference endpoint powered by the DeepSparse
```

**Comment by rahul-tuli** [comment]
A few nits then looks good!

File: examples/aws-sagemaker/README.md (hunk 3 above)
Comment (jeanniefinks) [reply examples/aws-sagemaker/README.md:36]: done!

**Review by rahul-tuli — APPROVED** [review]
Good Work @jeanniefinks!

**Review by rahul-tuli — APPROVED** [review]

**Review by bfineran — APPROVED** [review]

**Pull request merged**